inv-management/src/Main.java

```diff
@@ -0,0 +1,8 @@
+public class Main {
+    public static void main(String[] args) throws Exception {
+        Inventory bag = new Inventory();
+        bag.chooseTask();
```

Comment:
The main method should be responsible for driving the program not `Inventory`.  The inventory just knows about keeping its inventory while main is really pushing the user interface questions.  This also eliminates the need to call the list display in every single if/else statement.